zigbeegeneric/integrationpluginzigbeegeneric.json

```diff
@@ -471,7 +471,7 @@
                     "id": "ca9af6cf-2d15-4d54-ba07-3d2ce03445b8",
                     "name": "thermostat",
                     "displayName": "Zigbee Thermostat",
-                    "createMethods": ["auto"],
+                    "createMethods": [ "Auto" ],
```

Comment:
the only create method where you changed from `auto` to `Auto`, maybe align with the others? It should work anyways...